frontend-react/src/components/annotate.js

```diff
@@ -41,17 +42,30 @@ class Annotate extends Component {
         }
 
         return (
-            <div className="backgroundAll">
+            <div className="border">
                 <form className="selectForm" onSubmit={this._handleSubmit}>
                     <input type="file" id="file" className="fileInputButton" onChange={this._handleImageChange} />
                     <label for="file" class="fileLabel">Select Image</label>
-                    {/* <button type="submit" onClick={this._handleSubmit}>Upload Image</button> */}
+                    {/* <button type="submit" onClick={this._handleSubmit}>Send Annotations</button> */}
                 </form>
                 <div className="imageDisplay">
+
+                    {/* Currently has hardcoded images, needs to take in uploaded images */}
+                    
+                    {/* <ReactImageAnnotate
+                        selectedImage="https://i.imgur.com/SY4z4Cx.jpg"
+                        taskDescription="# Draw region around each Kiwifruit bunch"
+                        images={[{ src: "https://i.imgur.com/SY4z4Cx.jpg", name: "Kiwifruit.png" }]}
+                        regionClsList={["Kiwifruit", "Human Face"]}
+                        enabledTools="create-polygon, create-box"
+                        // onExit = {console.log(ReactImageAnnotate.MainLayoutState)}
+                    /> */}
+
                     {$imagePreview}
+
                 </div>
             </div>
-        );
+        )
```

Comment:
```suggestion
        );
```
Not sure why this was removed, but we should use semicolons to prevent bugs appearing later.